tox.ini

```diff
@@ -1,76 +1,124 @@
 [tox]
-envlist = py{36,37,38}-{offline},online,build_docs,figure
-isolated_build = True
+envlist =
+    py{36,37,38}-test{,-oldestdeps,-devdeps,-online}
```

Comment:
are we doing anything with `-test`? Is is clearer to have `py38-test` than just `py38`?

Reply:
Nope, i can remove it.

Reply:
Removed.